.github/workflows/sphinx_docs_to_gh_pages.yaml

```diff
@@ -3,8 +3,8 @@ name: Sphinx docs to gh-pages
 
 on:
   push:
-    branches:
-      - master
+    tags:
```

Comment:
Makes sense - I guess we'd rather have docs sync'd with the pypi release than with the latest GitHub.